### PR TITLE
Updated test cases of power_sum.py

### PR DESCRIPTION
**Pull request by Muhammadummerr**
### Describe your change:

I reduced the test cases, so it will run faster than before

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by Muhammadummerr** [comment]
The issue remains unresolved.

**Comment by Muhammadummerr** [comment]
power_sum.py has been removed from that list :)

**Comment by Muhammadummerr** [comment]
@cclauss

File: backtracking/power_sum.py [hunk 1]
```diff
@@ -90,4 +90,5 @@ def solve(needed_sum: int, power: int) -> int:
 if __name__ == "__main__":
     import doctest
 
+    # print(backtrack(80, 1, 1, 0, 0))
```
Comment (tianyizheng02) [inline backtracking/power_sum.py:93]: Do you plan to keep this line of code?

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]
Could you also modify the code to use the `**` operator instead of the `pow` function from `math`?

**Comment by tianyizheng02** [comment]
Also, please put the issue number in your PR's description:
> - [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

This way, GitHub can automatically link your PR to the original issue. However, don't use the word "fixes", because your PR doesn't completely fix the issue on its own.

**Comment by cclauss** [comment]
On your computer, please run:
`time python3 -m doctest backtracking/power_sum.py`
on both the new code and the old code so we know how much faster the new code is.

**Comment by Muhammadummerr** [comment]
**OLD CODE**
TotalSeconds      : 2.757677
TotalMilliseconds : 2757.677

**NEW CODE**
TotalSeconds      : 1.067801
TotalMilliseconds : 1067.801

**Review by tianyizheng02 — APPROVED** [review]

File: backtracking/power_sum.py [hunk 2]
```diff
@@ -19,25 +17,25 @@ def backtrack(
...
+    >>> backtrack(20, 1, 1, 0, 0)
+    (0, 64)
     """
     if current_sum == needed_sum:
         # If the sum of the powers is equal to needed_sum, then we have a solution.
         solutions_count += 1
         return current_sum, solutions_count
 
-    i_to_n = int(pow(current_number, power))
+    i_to_n = int(current_number**power)
```
Comment (tianyizheng02) [inline backtracking/power_sum.py:38]:
```suggestion
    i_to_n = current_number ** power
```
Removing unnecessary `int` cast (the variables are already ints)

**Review by tianyizheng02 — APPROVED** [review]

**Comment by tianyizheng02** [comment]
Contributes to #9718

**Pull request merged**